etc/templates/boot/install.erb

```diff
@@ -3,9 +3,17 @@ PROMPT 0
 MENU TITLE PXE Menu
 TIMEOUT 5
 TOTALTIMEOUT 5
+<% if firstboot %>
```

Comment:
Similarly to below, this could be `ONTIMEOUT <% firstboot ? 'INSTALL' : 'LOCAL' %>`